src/display/display_utils.js

```diff
@@ -65,11 +65,16 @@ class BaseCanvasFactory {
 }
 
 class DOMCanvasFactory extends BaseCanvasFactory {
+  constructor(ownerDocument = globalThis.document) {
```

Comment:
This needs to be future-proof, to avoid issues if/when we need to add more parameters here; hence please change to
```suggestion
  constructor({ ownerDocument = globalThis.document, } = {}) {
```
and update the call-site in `src/display/api.js` accordingly.